storm-client/src/jvm/org/apache/storm/task/TopologyContext.java

```diff
@@ -57,13 +57,14 @@
     private Map<String, Object> _executorData;
     private Map<Integer,Map<Integer, Map<String, IMetric>>> _registeredMetrics;
     private AtomicBoolean _openOrPrepareWasCalled;
-
+    private Map<String, Long> _blobToLastKnownVersion;
```

Comment:
Can we remove the `_` at the beginning.  I know it matches the rest of the file but we are moving away from this style.